lib/session.js

```diff
@@ -59,7 +59,7 @@ function storeData (input, data) {
     let val = input[i]
 
     // Delete values when users unselect checkboxes
-    if (val === '_unchecked' || val === ['_unchecked']) {
+    if ([val].flat().every((item) => item === '_unchecked')) {
```

Comment:
How confident are you in this change in behaviour? Might there be knock on effects of arrays of unchecked checkboxes now being `undefined` rather than an empty array?

Given it looks like there's code to filter arrays below (line 69), should we just remove the second part of the if condition and just do `val === '_unchecked'`, preserving the existing behaviour but removing the unreachable code?

Reply:
Yeah that's fair, fixing the old `undefined` behaviour could be unexpected

There are probably prototypes expecting array methods to work